tests/perf/service_invocation_http/service_invocation_http_test.go

```diff
@@ -122,8 +123,37 @@ func TestServiceInvocationHTTPPerformance(t *testing.T) {
 		t.Logf("added latency for %s percentile: %sms", v, fmt.Sprintf("%.2f", latency))
 	}
 
+	err = saveResults(baselineResult, daprResult)
+	if err != nil {
+		t.Error(err)
+	}
+
 	require.Equal(t, 0, daprResult.RetCodes.Num400)
 	require.Equal(t, 0, daprResult.RetCodes.Num500)
 	require.Equal(t, 0, restarts)
 	require.True(t, daprResult.ActualQPS > float64(p.QPS)*0.99)
 }
+
+func saveResults(baseline, dapr perf.TestResult) error {
+	l := []perf.TestResult{
+		baseline,
+		dapr,
+	}
+
+	accountName, accountKey := os.Getenv("AZURE_STORAGE_ACCOUNT"), os.Getenv("AZURE_STORAGE_ACCESS_KEY")
+	if len(accountName) > 0 && len(accountKey) > 0 {
+		now := time.Now().UTC()
+		y := now.Year()
+		m := now.Month()
+		d := now.Day()
+
+		container := fmt.Sprintf("%v-%v-%v", int(m), d, y)
```

Comment:
Can we have a folder structure like: YYYY/MM/DD and the file to be the testname-hh-mm-ss.

Reply:
azure blob storage doesn't support `/` in container names. right now it's `MM-DD-YYYY`. I can change the file name to `testname-hh-mm-ss`

Reply:
I was referring to using a single container and separate via folder structure instead. I am OK with one container per day.